.github/workflows/schedule_candidate_release.yml

```diff
@@ -13,21 +13,22 @@ jobs:
     # Don't run this in everyone's forks.
     if: github.repository == 'iree-org/iree'
     steps:
-      - name: Get the last green commit
-        id: last_green_commit
-        uses: talentpair/last-green-commit-action@8b4b3bcd4ab5d9ab16875ce20ae012c4ce47ae68
-        with:
-          github-token: ${{ secrets.GITHUB_TOKEN }}
-
       - name: Checking out repository
         uses: actions/checkout@e2f20e631ae6d7dd3b768f56a5d2af784dd54791 # v2.5.0
         with:
           token: ${{ secrets.WRITE_ACCESS_TOKEN }}
-          ref: ${{ steps.last_green_commit.outputs.result }}
+          fetch-depth: 0
+
+      - name: Checkout the most recent green commit
```

Comment:
Need to output a variable to be used at L52 and L64

Reply:
Thanks for catching! Fixed